config/deploy/values.yaml

```diff
@@ -1,3 +1,12 @@
 appConfig:
   rackEnv: "production"
   railsEnv: "production"
+resources:
```

Comment:
Why not leave this constant across all environments.

Reply:
because it doesn't use anything and we want to have the resources allocation as real as possible so we can use our resources the best way possible. And since they want to increase from 3 review envs to 10. This would make a big different in the resource allocation.

Reply:
> What about resources consumed during upgrades? I don't see any such peaks in the charts. Grab a longer period?

Even during the spin up of the environment it doesn't use much.
<img width="1612" alt="Screen Shot 2020-08-28 at 3 40 01 PM" src="https://user-images.githubusercontent.com/17147375/91604389-ec50fd00-e944-11ea-9292-ef30f9c00198.png">

Reply:
other dashboard as example:
<img width="1618" alt="Screen Shot 2020-08-28 at 3 47 24 PM" src="https://user-images.githubusercontent.com/17147375/91604877-da238e80-e945-11ea-993b-930fd10afebe.png">

Reply:
> because it doesn't use anything and we want to have the resources allocation as real as possible so we can use our resources the best way possible. And since they want to increase from 3 review envs to 10. This would make a big different in the resource allocation.

But does production really use more?

Reply:
ohh okay. Now I understood your point. I'm gonna check that.

Reply:
yeah. Production doesn't use that either. We can definitely decrease in all envs. I'm gonna adjust the PR @benlangfeld .
<img width="1623" alt="Screen Shot 2020-08-28 at 4 05 23 PM" src="https://user- images.githubusercontent.com/17147375/91606577-b0b83200-e948-11ea-8b3e-932a87be6ec0.png">
<img width="1607" alt="Screen Shot 2020-08-28 at 4 05 17 PM" src="https://user-images.githubusercontent.com/17147375/91606590-b4e44f80-e948-11ea-931f-a9ff41d4cc1b.png">